thirdeye/thirdeye-frontend/app/pods/manage/alert/route.js

```diff
@@ -16,36 +16,40 @@ const queryParamsConfig = {
 
 export default Route.extend({
   queryParams: {
-    replayId: queryParamsConfig
+    jobId: queryParamsConfig
   },
 
   beforeModel(transition) {
     const id = transition.params['manage.alert'].alertId;
-    const replayId = transition.queryParams.replayId;
+    const { jobId, functionName } = transition.queryParams;
     const durationDefault = '1m';
     const startDateDefault = buildDateEod(1, 'month').valueOf();
     const endDateDefault = buildDateEod(1, 'day');
 
     // Enter default 'explore' route with defaults loaded in URI
-    if (transition.targetName === 'manage.alert.index') {
+    // An alert Id of 0 means there is an alert creation error to display
```

Comment:
It's not very clear why an id of 0 means an error here. Is this an implementation choice?

Reply:
Yes... currently we have a deficiency in this API. It is not yet returning an ID for the new alert function. Once we receive success status from the create call, if we cannot confirm the existence of the alert in our DB (using `/data/autocomplete/functionByName`, I'm passing `0` as Id. I will change this to `-1`.